src/style.css

```
@@ -1 +1,42 @@
-/* write your styles here */
+body {
+  margin: 0;
+}
+
+#header,
+#footer,
+.important,
+.interesting,
+.surprising {
+  width: 300px;
+  height: 100px;
+  margin: auto;
+  transition: 0.5s;
```

```suggestion
  transition: height 0.5s;
```
define transition property